packages/indexer/indexer/chain_mapper.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass, field
+from typing import Dict, List
+
+
+@dataclass
+class CosmosChain:
+    chain_id: str
+    min_block_height: int
+    apis: List["CosmosAPI"] = field(default_factory=list)
+    current_api_index: int = 0
+
+
+@dataclass
+class CosmosAPI:
```

Comment:
I think we need to rename these

Reply:
to what?

Reply:
Not sure, but `CosmosAPI` isn't accurate.